### DISCUSSION TITLE
no livestream on edge and chrome

### DISCUSSION THREAD

**Discussion by nyunity** [jomjol/AI-on-the-edge-device#3005] [Q&A] [answered]
Hi,
I tried to reinstall my gasmeter, bought new sdcard and esp module... I tried two days before I checked, that my browser is the problem, not the esp itself.

On my smartphone, I can see the livestream at the startconfiguration, at my laptop not.

Does anyone know, which privileges I have to enable? I thought there is already everything enabled.

thx

**Comment by caco3** [comment]
There are no extra permissions required!
I also just tested with chrome and had no issues.
Can you try another browser like Firefox?

**Reply by nyunity** [reply, marked as answer]
I tried several browsers, I found the issue... my sophos client blocked it... tooked the IP from AIO into sophos. Now it works
